src/uk/gov/defra/ffc/Utils.groovy

```diff
@@ -118,4 +118,24 @@ class Utils implements Serializable {
     return ctx.sh(returnStdout: true, script:"curl -s -w \"%{http_code}\\n\" $url -o /dev/null").trim()
   }
 
+  static def getProvisionedQueueConfigValues(ctx, repoName, pr) {
+    def configValues = [:]
+    def filePath = 'provision.azure.yaml'
```

Comment:
In other places in the library there is `./` before a filename in the unix style to explicitly state the current folder. I think it works either way, but might be worth changing for consistency.